app/services/bolognese/writers/ris_writer_behavior.rb

```diff
@@ -0,0 +1,88 @@
+               "Collection Thesis (doctoral)"],
+        MAP: ["Map or Cartographic Material", "GenericWork Cartographic material",
+              "Collection Cartographic material"],
+        CHAP: ["Part of Book", "BookContribution default Book chapter", "BookContribution Book editorial",
+               "Collection Book chapter", "Collection Book editorial"],
+        CONF: ["Poster", "ConferenceItem Abstract", "ConferenceItem Conference paper (published)",
+               "ConferenceItem default Conference paper (unpublished)",
+               "ConferenceItem Conference poster (published)", "ConferenceItem Conference poster (unpublished)",
+               "ConferenceItem Lecture", "ConferenceItem Presentation", "Collection Conference paper (published)",
+               "Collection Conference paper (unpublished)", "Collection Conference poster (published)",
+               "Collection Conference poster (unpublished)"],
+        RPRT: ["Report", "Report Policy report", "Report default Research report", "Report Technical report",
+               "Collection Policy report", "Collection Research report", "Collection Technical report"],
+        COMP: ["Software or Program Code", "GenericWork Software", "Collection Software", "Dataset Software"],
+        VIDEO: ["Video", "TimeBasedMedia Animation", "TimeBasedMedia Video"],
+        MGZN: ["Article Magazine article", "Collection Magazine article"],
+        NEWS: ["Article Newspaper article", "Collection Newspaper article"],
+        MUSIC: ["GenericWork Musical notation", "Collection Musical notation"],
+        PAT: ["GenericWork Patent", "Collection Patent"],
+        ELEC: ["GenericWork Website", "Collection Website"],
+        BLOG: ["GenericWork Blog post", "Collection Blog post"]
+      }.freeze
+
+      included do
+        def ris
+          {
+            TY: calculate_resource_type(types),
+            T1: parse_attributes(titles, content: "title", first: true),
+            T2: container && container["title"],
+            AU: to_ris(creators),
+            DO: doi,
+            ED: meta.dig("editor"),
+            AB: parse_attributes(descriptions, content: "description", first: true),
+            KW: Array.wrap(subjects).map { |k| parse_attributes(k, content: "subject", first: true) }.presence,
+            PY: publication_year,
+            PB: publisher,
+            PP: meta.dig("place_of_publication"),
+            EP: container.to_h["lastPage"],
+            SN: ordered_identifiers,
+            JO: meta.dig("journal_title"),
+            LA: meta.dig("language"),
+            N1: meta.dig("add_info"),
+            UR: meta.dig("official_link"),
+            IS: container.to_h["issue"],
+            VL: container.to_h["volume"],
+            ER: ""
+          }
+            .compact
+            .map { |k, v| v.is_a?(Array) ? v.map { |vi| "#{k}  - #{vi}" }.join("\r\n") : "#{k}  - #{v}" }
```

Comment:
What do you guys think on moving this compact and custom mapping into a small method with a proper name? It would explains what it does and avoid spending 2 mins figuring it out what it does 6 months from now.